README.md

```diff
@@ -191,6 +192,24 @@ There are various tools to look XUIDs up online and they are also printed to the
 -e ALLOW_LIST_USERS="player1:1234567890,player2:0987654321"
 ```
 
+## Variables
+
+Custom server variables are passed in just like the allowlist or as a full JSON string.
```

Comment:
It would be good to somehow include here the link to the Microsoft docs you mentioned in your reply.

Reply:
Can do.